server/cmd/dcrdex/config.go

```diff
@@ -509,6 +511,7 @@ func loadConfig() (*dexConf, *procOpts, error) {
 		RegFeeConfirms:   cfg.RegFeeConfirms,
 		RegFeeXPub:       cfg.RegFeeXPub,
 		CancelThreshold:  cfg.CancelThreshold,
+		Anarchy:          cfg.Anarchy,
```

Comment:
I'd feel better if we checked that this isn't being used on mainnet. Probably an `os.Exit` but at least a big loud error message.

Reply:
Sure, will do.